diff --git a/build/ExpoConfigLoader.js b/build/ExpoConfigLoader.js
index 5145383f91383b4e0f29896136ec609836b89269..a71f139aa056193d43abe875424e6137fa927962 100644
--- a/build/ExpoConfigLoader.js
+++ b/build/ExpoConfigLoader.js
@@ -157,13 +157,13 @@ function installModuleCaptureHook() {
     const moduleProto = _module().default.prototype;
     const capturedModules = [];
     const originalCompile = moduleProto._compile;
-    moduleProto._compile = function(content, filename) {
+    moduleProto._compile = function(content, filename, ...rest) {
         capturedModules.push({
             id: this.id ?? filename,
             filename,
             content
         });
-        return originalCompile.call(this, content, filename);
+        return originalCompile.call(this, content, filename, ...rest);
     };
     return {
         getCapturedModules: ()=>capturedModules,
